milestone2.ts

```diff
@@ -138,24 +158,30 @@ type UpdateVolunteer = (
   vol: Volunteer,
   updates: Partial<Volunteer>
 ) => Volunteer;
-let updateVolunteer: UpdateVolunteer; // Code here
+let updateVolunteer: UpdateVolunteer = (vol, updates) => {
+  return {
+    ...vol,
+    ...updates
+  }
 
-// let updatedLocation = { city: "Seattle", state: "WA" };
-// let kyleNew = updateVolunteer(kyleClone, updatedLocation);
-//console.log(kyleNew);
+}
+
+let updatedLocation = { city: "Seattle", state: "WA" };
+let kyleNew = updateVolunteer(kyleClone, updatedLocation);
+console.log(kyleNew);
 
 // Question 7: Object Destructuring
 /* Now that we have our updated kyle, lets use object destructuring to get his name, age, and city
  */
 
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let name, age, city; // Code here
+  let name = vol.name, age = vol.age, city = vol.city; // Code here
```

Comment:
Can you resubmit with Object Destructuring in this question? If you're confused what object destructuring is you can find an example in the demo of [bootcamp session 3](https://h4i.notion.site/Intro-to-Javascript-83f29480d7c14b0c9163212b96f6ab0f)